server/views/pages/create/checkAnswers.njk

```diff
@@ -78,7 +80,9 @@
                 {{ inductionMeetingDetails(licence, isInProgress, validationErrors) }}
             {% endif %}
 
-            {% if licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS' %}
+
+            {% if (not licence.isInPssPeriod or (licence.isInPssPeriod and (not isInVariation or not licence.isActivatedInPssPeriod))) and
```

Comment:
I think this logic can be tidied up to
```
if (licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS' ) and (not licence.isInPssPeriod or not isInVariation or not licence.isActivatedInPssPeriod)
```

Having the AP/AP_PSS check first makes the logic more readable in my opinion, and the check of `(not licence.isInPssPeriod` makes the `licence.isInPssPeriod` part of the `and` irrelevant, because it will either be:
licence.isInPssPeriod is false, which means the logic will short-circuit at `not licence.isInPssPeriod` as it returns `true` in an `or`
The check is true, in which case the logic will be `true and ...`, at which point the `true` is irrelevant and it just relies on the other half of the statement. I'm happy to have a chat about this and if I'm missing something I can retract this statement!

Reply:
Alternatively, you might be able to grab the logic you use later to display the `expired conditions` section and just stick a `not` in front of it, ie:

`if (licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS') and not (isInPssPeriod and (isInVariation or licence.isActivatedInPssPeriod))`

I haven't thought as much about this one though so I might be missing something

Reply:
I don't see any difference between my logic and this if (licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS') and not (isInPssPeriod and (isInVariation or licence.isActivatedInPssPeriod)). Can you explain me what is the difference?

Reply:
From a purely code perspective, the only change is removing the `licence.isInPssPeriod` from `or (licence.isInPssPeriod and (...` as I'm not sure it's necessary.
Personally I thought it made the code more readable, but the logic should be unchanged as that looks fine to me. This is one of the things I'd like to discuss with other devs as if they disagree with me then I'm more than happy to back down!